src/lib.rs

```diff
@@ -298,11 +300,30 @@ impl<'a, T> MutSignals<'a, T> {
     }
 }
 
+/// Gets the libopus version string.
+///
+/// Applications may look for the substring "-fixed" in the version string to
+/// determine whether they have a fixed-point or floating-point build at runtime.
+pub fn version() -> &'static str {
+    // SAFETY: The pointer given from the `opus_get_version_string` function will be valid
```

Comment:
```suggestion
    // The pointer given from the `opus_get_version_string` function will be valid
```

Reply:
It's okay, since the doc-item is on top of `unsafe`, we do not need to add the safety-remark. Thanks for documenting it :)